.github/workflows/build.yaml

```diff
@@ -299,3 +369,37 @@ jobs:
           name: E2E report
           path: |
             src/security-server/e2e-test/build/allure-report/
+      - name: Load test report history
+        uses: actions/checkout@v4
+        if: always()
+        continue-on-error: true
+        with:
+          ref: allure-gh-pages
+          path: allure-reports
+
+      - name: Build test report
+        uses: simple-elf/allure-report-action@v1.9
+        if: always()
+        with:
+          gh_pages: allure-reports
+          subfolder: e2e
+          allure_history: allure-history
+          allure_results: src/security-server/e2e-test/build/allure-results
+      - name: Publish test report
+        uses: peaceiris/actions-gh-pages@v4
+        if: always()
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: allure-gh-pages
+          publish_dir: allure-history
+      - name: Add link to the report as a comment 
+        uses: actions/github-script@v7
+        if: always()
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'E2E Allure report: https://nordic-institute.github.io/X-Road/e2e/${{ github.run_number }}'
```

Comment:
perhaps opening in a new tab would be preferred?

Reply:
Based on what I found searching the topic it seems that this is not possible in GitHub comments.